package.json

```diff
@@ -58,7 +58,7 @@
     "ember-cli-htmlbars": "^6.1.1",
     "ember-cli-typescript": "^5.2.1",
     "ember-element-helper": "^0.6.1",
-    "ember-modifier": "^4.0.0",
```

Comment:
I imagine that listing `ember-modifier@v4` will be a breaking change for production apps, because teams will need time to address [deprecation warnings from `v3`](https://github.com/ember-modifier/ember-modifier/blob/v4.0.0-beta.1/MIGRATIONS.md).

I'll update `ember-modifier` to `v4` later (definitely after `ember-container-query@v3.0.0` is released).